pkg/parser/extended_syntax_parser.y

```diff
@@ -436,20 +436,19 @@ func (s StandardSelect) String() string {
 
 var mu sync.Mutex // Protect the use of global variable parseResult.
 
-func parseSQLFlowStmt(s string) (r *SQLFlowSelectStmt, e error) {
+func parseSQLFlowStmt(s string) (r *SQLFlowSelectStmt, idx int, e error) {
 	defer func() {
 		if r := recover(); r != nil {
-			var ok bool
-			e, ok = r.(error)
-			if !ok {
-				e = fmt.Errorf("%v", r)
-			}
+                        l := r.(*lexer)
+                        idx = l.pos - len(l.recent)
+			e = fmt.Errorf("%v", r)
```

Comment:
Here are some indentation issues. I am wondering why the `go fmt` in CI doesn't catch it.

Reply:
Good catch!  This is not a .go file, but a .y file. I will re-indent manually after I fix the CI failure.